planetiler-custommap/src/test/java/com/onthegomap/planetiler/custommap/TestAreaSourceFeature.java

```diff
@@ -0,0 +1,40 @@
+package com.onthegomap.planetiler.custommap;
+
+import com.onthegomap.planetiler.geo.GeometryException;
+import com.onthegomap.planetiler.reader.SourceFeature;
+import java.util.Collections;
+import java.util.Map;
+import org.locationtech.jts.geom.Geometry;
+
+public class TestAreaSourceFeature extends SourceFeature {
```

Comment:
Instead of `TestAreaSourceFeature` and `TestLinestringSourceFeature` you use the built-in `SimpleFeature` factory methods, something like:

```java
SimpleFeature.createFakeOsmFeature(newLineString(0, 0, 1, 1), tags, "osm", null, 1, List.of())
```

or

```java
SimpleFeature.createFakeOsmFeature(newPolygon(0, 0, 1, 0, 1, 1, 0, 0), tags, "osm", null, 1, List.of())
```

Reply:
Fixed in d110150